desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -689,6 +685,36 @@ public void updateStatusLabel() {
         _executionAlternativesButton.setEnabled(executeable);
     }
 
+    /**
+     * Checks if a job including its child jobs are correctly configured.
+     **/
+    private void checkAnalyzerJobsAreConfigured() {
```

Comment:
I see some code duplication here ... Could we push this logic down to the AnalysisJobBuilder instead of having it duplicated in the UI? I think the validate method that we had before should be a good place to have it.